prow.go

```diff
@@ -50,7 +50,7 @@ var supportedUpgradeTests = []string{"e2e-upgrade", "e2e-upgrade-all", "e2e-upgr
 
 // supportedPlatforms requires a job within the release periodics that can launch a
 // cluster that has the label job-env: platform-name.
-var supportedPlatforms = []string{"aws", "gcp", "azure", "vsphere", "metal"}
+var supportedPlatforms = []string{"aws", "gcp", "azure", "vsphere", "metal", "osd-aws"}
```

Comment:
hrm, i think osd should just be a platform, and you should be able to say `ows,osd` or just `osd` if you want it.

Reply:
Perhaps:
```
var supportedPlatforms = []string{"aws", ..., "osd"}
var supportedParameters = []string{"ovn", "proxy", "compact", "fips", ..., "osd-aws"}
```
This would allow a logic branch in the `osd-create-create` step to handle the variants versus a new (nearly identical) periodic if `aws,osd` is a separate item in `supportedPlatforms`.

Reply:
Current commit has `osd` as a platform and `osd-aws` as a variant (anticipating variants like `osd-gcp` in the future).